pkg/recommender.py

```diff
@@ -21,7 +21,7 @@
 POLL_TIME = 120
 MAX_MESSAGES = 4
 MESSAGES_SENT_TODAY = 0
-COOLDOWN_TIME = 1800 #30 min
+COOLDOWN_TIME = 300 #5 min
```

Comment:
should we update this? 5 mins is for demo purpose only right?

Reply:
oh sure, should we set it at 30 minutes?

Reply:
yes, keep it as 30 mins. if we need to make it easier to test/change, we can study @kaiyingshan 's design. make it an argument whose default value is 30 mins but you can set the value you like in the test script

Reply:
got it, I changed it to 30 minutes